providers/vsphere/provider/provider_test.go

```diff
@@ -164,5 +153,5 @@ func TestVsphereDiscovery(t *testing.T) {
 	}, nil)
 	require.NoError(t, err)
 	assert.NotNil(t, resp.Asset)
-	assert.Equal(t, 8, len(resp.Inventory.Spec.Assets)) // api + esx + vm
+	// assert.Equal(t, 8, len(resp.Inventory.Spec.Assets)) // api + esx + vm
```

Comment:
nit: shouldn't this be 1, when only discovering the API?

Reply:
apparently not. That is something @chris-rock suggested because this test wasn't working initially